### DISCUSSION TITLE
PR/Issue preview image generator only processes the first markdown tag

### DISCUSSION THREAD

**Discussion by TellowKrinkle** [community/community#23435] [Projects and Issues]
<p>To reproduce:</p>
<ul>
<li>Post <a href="https://github.com/rtissera/libchdr/issues/55" rel="noopener nofollow ugc">https://github.com/rtissera/libchdr/issues/55</a> in an application that pulls site preview thingies from GitHub (e.g. Discord)</li>
<li>Look at the generated image: <a href="https://opengraph.githubassets.com/291935119dd64e6a9dbdf53839ac5e69be1640c1e2e0f7143d7bcc047c0eae4c/rtissera/libchdr/issues/55" rel="noopener nofollow ugc">https://opengraph.githubassets.com/291935119dd64e6a9dbdf53839ac5e69be1640c1e2e0f7143d7bcc047c0eae4c/rtissera/libchdr/issues/55</a>
</li>
</ul>
<p>Expected result:<br>
Both pieces of text wrapped in ``s are rendered <code>like this</code></p>
<p>Actual result:<br>
The first piece of text wrapped in ``s is rendered <code>like this</code> but the second is rendered `like this`</p>

**Comment by jainamoswal** [comment]
<p>I think it supports multiple Markdown tags now as I can’t see the unwrapped anymore.</p>
